README.md

```diff
@@ -149,6 +149,8 @@ _Generic low-level UI components for building apps._
 
 _Components and libraries to interact with backend services._
 
+* 🔴 [**FxA**](components/service/fxa/README.md) - A library for integrating with Firefox Accounts.
```

Comment:
Even though the folks working on this know what FxA is: Can we name this more clearly? Maybe just FirefoxAccounts (service-firefox-accounts)?